tests/Dispatching/DispatchingSetTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace EquipTests\Dispatching;
+
+use Auryn\Injector;
+use Equip\Adr\DomainInterface;
+use Equip\Directory;
+use Equip\Exception\DispatchingException;
+use Equip\Dispatching\DispatchingSet;
+use PHPUnit_Framework_TestCase as TestCase;
+
+class DispatchingSetTest extends TestCase
+{
+    public function testWithInvalidEntries()
+    {
+        $this->setExpectedExceptionRegExp(
+            DispatchingException::class,
+            '/Dispatch .* is not invokable/i'
+        );
+
+        new DispatchingSet(['foo']);
```

Comment:
While I would tend to doubt that a function named `foo()` will be defined within the test context in the future, an easy way to future-proof against it would be to use a class instance without `__invoke()` defined, such as `stdClass`.